routes/video.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const videoRouter = express.Router();
+const { getVideoLists } = require("../controllers/video");
+
+videoRouter.get("/", getVideoLists);
```

Comment:
다른 router들이랑 코드 통일을 위해 아래와 같이 작성하는게 좋을 것 같습니다.   

```suggestion
videoRouter
  .get("/", getVideoLists);
```

Reply:
현재 적용된 lint 설정으로 인해 변경이 불가하네요.

나중에 다른 method들이 추가되면 수정되지 않을까 생각됩니다!